tests/integration/test_schema.py

```diff
@@ -1096,30 +1094,42 @@ async def test_schema_types(registry_async_client, trail):
 
 
 @pytest.mark.parametrize("trail", ["", "/"])
-async def test_schema(registry_async_client, trail):
-    subject = new_random_name("subject")
-    schema_str = '{"type": "string"}'
+async def test_schema_repost(registry_async_client, trail):
```

Comment:
please always add typing when introducing new code (this applies to the other functions too)

Reply:
I'll add annotations to all functions in `test_schema.py`. It's nicer to have them in all, not just in the new ones.